tools/depends/target/Makefile

```diff
@@ -97,6 +97,7 @@ pythonmodule-pil: $(ZLIB) libjpeg-turbo libpng freetype2 python26
 libsdl2: $(LINUX_SYSTEM_LIBS)
 libxslt: libgcrypt
 ffmpeg: $(ICONV) $(ZLIB) bzip2 libvorbis $(FFMPEG_DEPENDS)
+libcec: platform
```

Comment:
add-ons will need to be updated indeed, like this https://github.com/kodi-pvr/pvr.hts/pull/24
will update the other ones probably in the evening. thought I had done this on my laptop already but not 100% sure. without this, they'll fail to build because the include dir changed. code stays the same.

you should only rebuild add-ons if their hash changes on github.